hw/ip/i2c/doc/_index.md

```diff
@@ -1,3 +1,223 @@
+
 ---
-title: "I2C Technical Specification" 
+title: "I2C HWIP Technical Specification"
 ---
+
+
+# Overview 
+
+This document specifies I2C hardware IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.](https://github.com/lowRISC/stwg-base/blob/master/doc/ComportabilitySpecification.md)
+See that document for integration overview within the broader top level system.
+
+
+## Features 
+
+- Two pin clock-data parallel bidirectional external interface 
+- The initial revision only supports I2C as a Host (I2C Master).
+- Support for standard (100 kbaud), fast (400 kbaud) and fast-plus (1 Mbaud) modes
+- Bandwidth up to 1 Mbaud
+- Support for all “Mandatory” features as specified for I2C Hosts (as listed in Table 2 of the I2C specification): 
+    - Start Condition
+    - Stop Condition
+    - Acknowledge
+    - 7-bit device address
+- Support for the following optional capabilities:
+    - 10-bit device addressing
+    - Clock stretching (may be required by some devices)
+- *No support at this time* for I2C Target (I2C Slave) functionality
```

Comment:
"I2C Slave" in quotes

Reply:
Also added second reference to same footnote